Hydra/Serializer/CollectionNormalizer.php

```diff
@@ -92,7 +100,7 @@ public function normalize($object, $format = null, array $context = array())
                 '@context' => $this->contextBuilder->getResourceContext($resource, $context),
                 '@id' => $context['request_uri'],
             ];
-            list($parts, $parameters) = $this->parseRequestUri($context['request_uri']);
+            $uri = $this->parseRequestUri($context['request_uri']);
```

Comment:
`http_build_query` always outputs integer index of array, e.g. `/dummies?relatedDummies[]=%2Frelated_dummies%2F1` becomes:

`/dummies?relatedDummies%5B%05D=%2Frelated_dummies%2F1` (actual)
`/dummies?relatedDummies%5B%5D=%2Frelated_dummies%2F1` (expected)